src/component/shared/sidebar/Sidebar.jsx

```diff
@@ -9,8 +11,15 @@ import { RiMovieFill } from "react-icons/ri";
 import { MdPlaylistPlay } from "react-icons/md";
 
 export const Sidebar = () => {
+  const { sidebarToggle } = useSelector((store) => store.displayTimeline);
+  const dispatch = useDispatch();
+
   return (
-    <aside className="hidden flex flex-col items-center basis-full md:basis-44 shrink-0 text-center bg-neutral-800">
+    <aside
+      className={`${
+        sidebarToggle ? "flex" : "hidden"
+      } md:flex flex-col items-center basis-full md:basis-44 shrink-0 text-center bg-neutral-800`}
+    >
       <NavLink
         to="/"
         className={({ isActive }) =>
```

Comment:
this is repeating one 
you can put this active state outside make function for this 
and pass in className

Reply:
this too